### PR TITLE
fix compiler warnings

### PR DESCRIPTION
**Pull request by Jojo-Schmitz**

### CODE REVIEW

File: mscore/scoretab.cpp [hunk 1]
```diff
@@ -167,12 +167,12 @@ TabScoreView* ScoreTab::tabScoreView(int idx)
             return nullptr;
             }
 
-      QVariant data = tab->tabData(idx);
-      IF_ASSERT_FAILED(data.isValid()) {
```
Comment (Jojo-Schmitz) [inline mscore/scoretab.cpp]: MSVC C4458

File: libmscore/cmd.cpp [hunk 2]
```diff
@@ -3239,21 +3239,21 @@ Segment* Score::setChord(Segment* segment, int track, Chord* chordTemplate, Frac
                   //and add the tie to score
                   const std::vector<Note*> notes = chord->notes();
                   if (addTie) {
-                        for (size_t i = 0; i < notes.size(); ++i) {
-                              tie[i]->setEndNote(notes[i]);
-                              notes[i]->setTieBack(tie[i]);
-                              undoAddElement(tie[i]);
```
Comment (Jojo-Schmitz) [inline libmscore/cmd.cpp]: MSCV C4456

File: libmscore/cmd.cpp [hunk 3]
```diff
@@ -3239,21 +3239,21 @@ Segment* Score::setChord(Segment* segment, int track, Chord* chordTemplate, Frac
...
                         addTie = false;
                         }
                   //if we're not the last element in the duration list,
                   //set tie forward
                   if (i+1 < n) {
-                        for (size_t i = 0; i < notes.size(); ++i) {
-                              tie[i] = new Tie(this);
-                              tie[i]->setStartNote(notes[i]);
-                              tie[i]->setTrack(track);
-                              notes[i]->setTieFor(tie[i]);
```
Comment (Jojo-Schmitz) [inline libmscore/cmd.cpp]: dito

File: mscore/prefsdialog.ui [hunk 4]
```diff
@@ -1347,7 +1347,7 @@
          <property name="checked">
           <bool>false</bool>
          </property>
-         <layout class="QGridLayout" name="gridLayout_4">
```
Comment (Jojo-Schmitz) [inline mscore/prefsdialog.ui]: duplicate name

File: mscore/scoretab.cpp [hunk 5]
```diff
@@ -167,12 +167,12 @@ TabScoreView* ScoreTab::tabScoreView(int idx)
...
+      QVariant dta = tab->tabData(idx);
```
Comment (anatoly-os) [inline mscore/scoretab.cpp:170]: Call it `tabData`. Better than `dta` :)

**Pull request merged**